src/server/routes/entity/edition.js

```diff
@@ -136,6 +136,7 @@ router.get('/create', auth.isAuthenticated, loadIdentifierTypes,
 				title: 'Add Edition',
 				heading: 'Create Edition',
 				subheading: 'Add a new Edition to BookBrainz',
+				script: 'edition',
```

Comment:
Also need to update the template name from `entity/create/<entity>` to `entity/create/create-common` for all entity create routes.